src/modules/ui.js

```diff
@@ -60,6 +60,18 @@ class CoreUiApi extends BaseApi {
     hideHelp() {
         super.sendMsgToPc('hideHelp');
     }
+
+    /**
+     * Show an interaction by ID.
+     *
+     * @example
+     * myClientApp.coreUi.showInteractionDetails('af2ef59d-9bc5-4436-8738-97c04869c81c');
+     *
+     * @since 1.1.0
+     */
+    showInteractionDetails(conversationId) {
```

Comment:
After moving this out to a separate module, would you mind adding a test for it?  You can follow lifecycleSpec.js to mock up the `_myParent` prop and assert the message is sent.